cmd/relay-runtime-tools/runner.go

```diff
@@ -49,11 +59,24 @@ func (rr *realRunner) Run(args ...string) error {
 	}
 
 	if mu != nil {
-		if err := getEnvironmentVariables(mu); err != nil {
+		err = retry.Wait(ctx, func(ctx context.Context) (bool, error) {
+			if err := getEnvironmentVariables(mu); err != nil {
+				return false, err
+			}
+
+			return true, nil
+		}, waitOptions...)
+		if err != nil {
 			log.Println(err)
 		}
 
-		if err := validateSchemas(mu); err != nil {
+		err = retry.Wait(ctx, func(ctx context.Context) (bool, error) {
+			if err := validateSchemas(mu); err != nil {
```

Comment:
May be worth noting that once schema validation becomes an actual error we'll have to distinguish between network errors and invalid schema errors here.

Reply:
I'm ambivalent about conflating validation with errors, but I might specifically avoid 422s in this code.